config/database.yml

```diff
@@ -1,25 +1,25 @@
-# SQLite. Versions 3.8.0 and up are supported.
-#   gem install sqlite3
+# Postgres. Versions 3.8.0 and up are supported.
```

Comment:
can we update this comment? 🙏 just to align everything

Reply:
Updated